examples/language_modeling_from_scratch-tf.ipynb

```diff
@@ -1198,26 +1218,28 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Cloning https://huggingface.co/Rocketknight1/bert-base-cased-finetuned-wikitext2 into local empty directory.\n"
+      "/home/matt/miniconda3/envs/tensorflow26/lib/python3.9/site-packages/huggingface_hub/hf_api.py:715: FutureWarning: `create_repo` now takes `token` as an optional positional argument. Be sure to adapt your code!\n",
+      "  warnings.warn(\n",
+      "/home/matt/PycharmProjects/notebooks/examples/mlm_from_scratch_model_save is already a clone of https://huggingface.co/Rocketknight1/bert-base-cased-finetuned-wikitext2. Make sure you pull the latest changes with `repo.git_pull()`.\n"
```

Comment:
Maybe some warnings like this one can be removed for clarity/readability

Reply:
Those warnings started being fired recently by the `PushToHubCallback`, although it still works correctly. Do you want me to leave merging this until I investigate that, or should I just merge this now and put that on the list to be fixed when I can? It's only cosmetic, as far as I can tell.

Reply:
You can merge it now!

Reply:
Sure!